pallets/staking/src/lib.rs

```diff
@@ -1208,7 +1208,7 @@ pub mod pallet {
                 ledger.active += extra;
                 // last check: the new active amount of ledger must be more than ED.
                 ensure!(
-                    ledger.active >= T::Currency::minimum_balance(),
+                    ledger.active >= <ExistentialDeposit<T>>::get(),
```

Comment:
https://github.com/paritytech/substrate/blob/master/frame/staking/src/pallet/mod.rs#L907
no need to change